ci-operator/step-registry/rosa/operator/cleanup/rosa-operator-cleanup-commands.sh

```
@@ -28,6 +42,38 @@ log "Cleaning up test operator resources"
 if oc get clusterpackage "${CLUSTER_PACKAGE_NAME}" &>/dev/null; then
     log "Deleting ClusterPackage ${CLUSTER_PACKAGE_NAME}"
     oc delete clusterpackage "${CLUSTER_PACKAGE_NAME}" --timeout=120s || true
+    # Wait for deletion to complete so CRDs are released
+    for i in $(seq 1 24); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)